fuel-relayer/src/validators.rs

```diff
@@ -26,16 +26,94 @@ impl Validators {
         self.set = db.get_validators().await;
     }
 
-    /// Get validator set
+    /// Get validator set for da_height.
     pub async fn get(
-        &mut self,
+        &self,
         da_height: DaBlockHeight,
+        db: &mut dyn RelayerDb,
     ) -> Option<HashMap<Address, (u64, Option<Address>)>> {
```

Comment:
[nit] improve readability by aliasing return types.

```suggestion
    ) -> Option<HashMap<StakingAddress, (u64, Option<ConsensusAddress>)>> {
```

Reply:
Maybe even create concrete types for these somewhat complex returns. Its a bit hard to read if we don't

Reply:
New types look a lot nicer and more readable, but I will create new branch to propagate this. My preference would be `StakingPublicKey` and `ConsensusPublicKey`. @Voxelot & @vlopes11 is this naming more aligned or does `StakingAddress` and `ConsensusAddress` sound better for you?

Reply:
Or to be more precise it is not `StakingAddress` but more like `ValidatorAddress` or `ValidatorPublicKey`

Reply:
I guess what I'm not sure about is whether we use the consensus address (32 bytes) or key (64 bytes) in the contracts etc. We could go really generic for now and just do the bare minimum to differentiate these vars, i.e. `ValidatorId` and `ConsensusSessionId`. I like SessionId because it illustrates the transitory nature of the key/address in relation to the ValidatorId.

Reply:
`Consensus` part is a public key but in the compressed form of 32bytes